From d164c35893039a0d310526e4a1bb7ccaaeaf91d6 Mon Sep 17 00:00:00 2001
Message-Id: <d164c35893039a0d310526e4a1bb7ccaaeaf91d6.1588111715.git.marcel.ziswiler@toradex.com>
In-Reply-To: <e468d4ece1c2f170254466333c90efcbb368da5d.1588111715.git.marcel.ziswiler@toradex.com>
References: <e468d4ece1c2f170254466333c90efcbb368da5d.1588111715.git.marcel.ziswiler@toradex.com>
From: Gerard Salvatella <gerard.salvatella@toradex.com>
Date: Fri, 23 Nov 2018 15:47:51 +0100
Subject: [PATCH 12/17] ARM: dts: imx7-colibri-eval-v3: disable PWM B/C/D

PWM is not really required by Toradex Easy Installer and might
interfere on some carrier boards. Disable them by default.

Signed-off-by: Gerard Salvatella <gerard.salvatella@toradex.com>
---
 arch/arm/boot/dts/imx7-colibri-eval-v3.dtsi | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm/boot/dts/imx7-colibri-eval-v3.dtsi b/arch/arm/boot/dts/imx7-colibri-eval-v3.dtsi
index d6a7deeda9356..b28c0f48ce738 100644
--- a/arch/arm/boot/dts/imx7-colibri-eval-v3.dtsi
+++ b/arch/arm/boot/dts/imx7-colibri-eval-v3.dtsi
@@ -221,18 +221,6 @@
 	status = "okay";
 };
 
-&pwm2 {
-	status = "okay";
-};
-
-&pwm3 {
-	status = "okay";
-};
-
-&pwm4 {
-	status = "okay";
-};
-
 &uart1 {
 	status = "okay";
 };
-- 
2.25.4

